kubetools/settings.py

```diff
@@ -23,6 +23,8 @@ class KubetoolsSettings(object):
 
     CRONJOBS_BATCH_API_VERSION = 'batch/v1'  # if k8s version < 1.21+ should be 'batch/v1beta1'
 
+    REGISTRY_CHECK_SCRIPT = None
```

Comment:
would this be easier to set as an ENV VAR? or is it trivial to override kubetools settings as needed?

Reply:
They're loaded from ` ~/.kubetools/kubetools.conf `.